### PR TITLE
adding sider

### PR DESCRIPTION
**Pull request by tulasikurete**
sider
image

### CODE REVIEW

File: src/app/components/Sider.tsx [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+
+                { value: "Discount", label: "Discount", disabled: true },
+              ]}
+            />
+          </Space>
+        </section>
+        <section>
+          <Divider />
+          <Space>
+            <Space>
```
Comment (Namahsivaya7) [inline src/app/components/Sider.tsx]: please remove duplicate "space" tag

File: src/app/components/Sider.tsx [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+  function callback() {
+    // console.log(key);
+  }
+
+  const { Panel } = Collapse;
+  return (
+    <main className="sideresponsive">
+      <h1>Products For You</h1>
+      <section>
+        <section>
```
Comment (Namahsivaya7) [inline src/app/components/Sider.tsx]: please remove duplicate tag

File: src/app/components/Sider.tsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+          </Flex>
+
+          <Collapse
+            expandIconPosition="right"
+            defaultActiveKey={["1"]}
+            onChange={callback}
+            style={{ borderTop: "1px solid grey", borderRadius: 0 }}
+            ghost
+          >
+            <Panel header={<Typography>color</Typography>} key="1">
```
Comment (Namahsivaya7) [inline src/app/components/Sider.tsx]: could you please remove "typography" tag which assign to header

**Review by Namahsivaya7 — CHANGES_REQUESTED** [review]

**Comment by tulasikurete** [comment]
image

File: src/app/components/Sider.tsx [hunk 4]
```diff
@@ -0,0 +1,149 @@
...
+
+export default function Sider() {
+  const handleChange = (value: string) => {
+    console.log(`selected ${value}`);
+  };
+  const onChange = (checkedValues: CheckboxValueType[]) => {
+    console.log("checked = ", checkedValues);
+  };
+  function callback() {
+    // console.log(key);
```
Comment (NagaDurga16) [inline src/app/components/Sider.tsx]: please remove this

File: src/app/components/Sider.tsx [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+  }
+
+  const { Panel } = Collapse;
+  return (
+    <main className="sideresponsive">
+      <h1>Products For You</h1>
+      <section>
```
Comment (NagaDurga16) [inline src/app/components/Sider.tsx]: please remove this sections

**Review by NagaDurga16 — CHANGES_REQUESTED** [review]

**Comment by tulasikurete** [comment]
image

**Review by NagaDurga16 — APPROVED** [review]

**Review by arajajyothibabu — APPROVED** [review]
LGTM, congrats on your first PR 🎉

**Pull request merged**